### PR TITLE
New pull request!

### PR DESCRIPTION
**Pull request by mcmikecreations**
My awesome pull request!

### CODE REVIEW

File: 1.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Crash Course
-Mykola Morozov code!
+Mykola Morozov new code!
```
Comment (SergiyIvan) [inline 1.txt:2]: Nice change!

File: 2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+My new code!
```
Comment (SergiyIvan) [inline 2.txt:1]: Good addition!

**Review by SergiyIvan — APPROVED** [review]
Great work, thank you!

File: 1.txt (hunk 1 above)
Comment (mcmikecreations) [reply 1.txt:2]: Cheers!

File: 2.txt (hunk 2 above)
Comment (mcmikecreations) [reply 2.txt:1]: Thank you!

**Pull request merged**